src/components/30-organisms/table-sortable/index.js

```diff
@@ -123,6 +121,29 @@ class AXATableSortable extends LitElement {
     return this.areLengthValuesConsistent(tbodyL, theadL, tfootL);
   }
 
+  sortByIndex(index, config) {
+    const sortingAria = this.getSortingAria(config);
+
+    if (sortingAria !== 'none') {
+      const sortAs = sortingAria === ASC ? DESC : ASC;
+      const tmpModel = { ...this.model };
```

Comment:
The use of a cloned tmpModel needs a clear comment. I believe I know why, namely because direct sorting on the model would cause rerender storms or something like this, as model is a reactive property.

Whatever the reason for cloning is, describe it clearly in a comment.